how-to/how-to-use-rootless-kata-with-podman.md

```diff
@@ -0,0 +1,108 @@
+# How to use Kata Containers with Podman rootlessly
```

Comment:
- I've raised https://github.com/kata-containers/tests/pull/1949 to allow the spell checker to accept "Podman".
- Please add a table of contents at the top of this doc. One method to create the text to paste into the doc:
  ```sh
  $ ../tests/cmd/check-markdown/kata-check-markdown toc how-to/how-to-use-rootless-kata-with-podman.md
  ```

Reply:
Thank you for adding Podman to the spell checker!